### PR TITLE
Added localstack batch and locale2e

### PR DESCRIPTION
**Pull request by jim-sheldon**
- Create S3 buckets
- Create Batch Job Queue
- Add email address to SES
- Curator AWS clients use localstack when using locale2e docker-compose stack
- Curator async download uses localstack when using locale2e docker-compose stack

### CODE REVIEW

File: dev/run_full_stack.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+# Store current directory.
+pushd `pwd`
+# We have to run docker compose from this directory for it to pick up the .env file.
+cd `dirname "$0"`
```
Comment (abhidg) [inline dev/run_full_stack.sh:6]: can be pushd `dirname "$0"` as pushd also changes directory

File: dev/setup_localstack.py [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+        try:
+            response = requests.get(healthcheck_url)
+            if BATCH_READY in response.text:
+                return
+        except requests.exceptions.ConnectionError:
+            pass
+        counter += 1
+        print("Waiting for localstack")
+        sleep(5)
+    raise Exception("Localstack not available")
```
Comment (iamleeg) [inline dev/setup_localstack.py:172]: When I try running `dev/run_full_stack.sh` locally I hit this exception, is there any setup I need to do? I've set `LOCALSTACK_API_KEY` in `dev/.env`.

File: dev/setup_localstack.py [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+        self.ses_client.verify_email_identity(
+            EmailAddress=email_address
+        )
+        print("Verified email address")
+
+
+def wait_for_localstack():
+    healthcheck_url = "".join([LOCALSTACK_URL, "/health"])
+    counter = 0
+    while True and counter < 15:
```
Comment (abhidg) [inline dev/setup_localstack.py:162]: same as `while counter < 15`

**Review by abhidg — COMMENTED** [review]
Couple of small changes, looks good otherwise!

File: dev/setup_localstack.py (hunk 3 above)
Comment (jim-sheldon) [reply dev/setup_localstack.py]: whoops, forgot to remove that during cleanup. good catch.

File: dev/run_full_stack.sh (hunk 1 above)
Comment (jim-sheldon) [reply dev/run_full_stack.sh]: oh, did not know that. updated.

File: dev/setup_localstack.py (hunk 2 above)
Comment (jim-sheldon) [reply dev/setup_localstack.py:172]: I suspect a timing issue. I increased the number of attempts to 20.

Comment (abhidg) [reply dev/setup_localstack.py:172]: Can't reproduce this, but I got this error message:

```Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /tmp/localstack```

Creating /tmp/localstack fixed this

Comment (iamleeg) [reply dev/setup_localstack.py:172]: > Creating /tmp/localstack fixed this

Should that ideally go in the host's `/tmp` or the docker image's `/tmp`? Either way, it needs to be scripted.

Comment (jim-sheldon) [reply dev/setup_localstack.py:172]: `/tmp` exists in the docker image, but I'm not using it for anything--the volume declared in the compose file was a default--so I removed it.

Comment (iamleeg) [reply dev/setup_localstack.py:172]: I still see that the timeout exception is raised before the localstack has launched:

```
setup-localstack_1  | Waiting for localstack
setup-localstack_1  | Traceback (most recent call last):
setup-localstack_1  |   File "./setup_localstack.py", line 177, in <module>
setup-localstack_1  |     wait_for_localstack()
setup-localstack_1  |   File "./setup_localstack.py", line 172, in wait_for_localstack
setup-localstack_1  |     raise Exception("Localstack not available")
setup-localstack_1  | Exception: Localstack not available
setup-localstack_1 exited with code 1
localstack_1        | Waiting for all LocalStack services to be ready
localstack_1        | 2021-07-16T08:25:19:INFO:botocore.credentials: Found credentials in environment variables.
localstack_1        | Waiting for all LocalStack services to be ready
localstack_1        | Ready.
localstack_1        | 2021-07-16 08:25:24,808:API: 127.0.0.1 - - [16/Jul/2021 08:25:24] "GET / HTTP/1.1" 200 -
localstack_1        | 2021-07-16T08:25:24:INFO:localstack.utils.analytics.profiler: Execution of "start_api_services" took 12558.80ms
```

Could the setup image use [`wait-for-it.sh`](https://github.com/vishnubob/wait-for-it/blob/master/wait-for-it.sh) and run when the localstack service is finally accepting connections?

**Review by abhidg — APPROVED** [review]

**Comment by iamleeg** [comment]
I see the same test failure in FF on a different PR. I'll try to fix and get a commit for you to cherry-pick.

**Comment by iamleeg** [comment]
I'm not proud but https://github.com/globaldothealth/list/pull/2003/commits/df56a02323a6d04372d94caddde09308b79ff3a9 works.

**Review by iamleeg — APPROVED** [review]

**Pull request merged**